src/stylesheet-registry.js

```diff
@@ -163,11 +183,17 @@ function invariant(condition, message) {
   }
 }
 
-export const StyleSheetContext = createContext(new StyleSheetRegistry())
+export const StyleSheetContext = createContext(null)
+
+export function createStyleRegistry() {
```

Comment:
Why an extra function as we already exported the class

Reply:
`StyleSheetRegistry` export is for testing. Every valid imports will come from index later.